types/global-agent/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/global-agent",
-    "version": "2.1.9999",
```

Comment:
This could probably be updated to `3.0.9999`.

Reply:
Done.